src/xcode/ENA.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -82,6 +82,15 @@
           "version": null
         }
       },
+      {
+        "package": "FMDB",
```

Comment:
Please revert package changes, we can't update without rigorously testing.

Reply:
this is already reverted